package.json

```diff
@@ -24,6 +24,8 @@
     "node": ">=10.0"
   },
   "dependencies": {
+    "@nomiclabs/buidler": "^1.0.1",
+    "@nomiclabs/buidler-ethers": "^1.0.1",
```

Comment:
Shall we add them as peer dependencies to reduce the amount of dependencies?

Reply:
We can, but these two buidler packages are far smaller than `ganache-core`. Is there really need for that?